pages/vi/vi-github-and-repositories.md

```diff
@@ -71,43 +74,43 @@ To fetch updates from the upstream repository, configure it as follows:
 
 1. Open your command prompt/terminal and navigate to the repository directory:
 
-  ```bash
-  cd <YourUserName>.github.io
-  ```
+```bash
+cd <YourUserName>.github.io
+```
 
 2. List the current configured remote repository:
 
-  ```bash
-  git remote -v
-  ```
+```bash
+git remote -v
+```
 
-  This should show:
+This should show:
 
-  ```bash
-  origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (fetch)
-  origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (push)
-  ```
+```bash
+origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (fetch)
+origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (push)
+```
```

Comment:
Are you using a linter that automatically applies rules when you save a file? I’m noticing changes unrelated to the issue, and comparing the two versions, the original one renders better with properly indented code blocks.

https://raw.githack.com/Gavinp14/Gavinp14.github.io/3369-push-and-fetch-definitions/index.html#!./pages/vi/vi-github-and-repositories.md#3._Configure_a_Remote_Repository_for_Your_Fork

https://open-learning-exchange.github.io/#!./pages/vi/vi-github-and-repositories.md#3._Configure_a_Remote_Repository_for_Your_Fork

Reply:
I will make these changes. I'm not sure why lines 77-92 are showing changes as I only added the push and fetch definitions.